Makefile

```diff
@@ -110,6 +110,8 @@ test-cli: test/assets
 #
 
 # Clone OCR-D/assets to ./repo/assets
+# FIXME does not work if already checked out
+# FIXME should be a proper (VCed) submodule
```

Comment:
true, we're using submodules in most places now.

Reply:
Let's address this another time.